src/main/java/org/assertj/core/api/AbstractLongAdderAssert.java

```diff
@@ -0,0 +1,552 @@
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.api;
+
+import org.assertj.core.data.Offset;
+import org.assertj.core.data.Percentage;
+import org.assertj.core.internal.ComparatorBasedComparisonStrategy;
+import org.assertj.core.internal.Longs;
+import org.assertj.core.util.CheckReturnValue;
+import org.assertj.core.util.VisibleForTesting;
+
+import java.util.Comparator;
+import java.util.concurrent.atomic.LongAdder;
+
+import static org.assertj.core.error.ShouldHaveValue.shouldHaveValue;
+import static org.assertj.core.error.ShouldNotContainValue.shouldNotContainValue;
+
+/**
+ * Base class for all implementations of assertions for {@link LongAdder}s.
+ *
+ * @param <SELF> the "self" type of this assertion class. Please read &quot;<a href="http://bit.ly/1IZIRcY"
+ *               target="_blank">Emulating 'self types' using Java Generics to simplify fluent API implementation</a>&quot;
+ *               for more details.
+ *
+ * @author Grzegorz Piwowarek
+ */
+class AbstractLongAdderAssert<SELF extends AbstractLongAdderAssert<SELF>> extends AbstractAssert<LongAdderAssert, LongAdder> implements NumberAssert<LongAdderAssert, Long> {
+
+  @VisibleForTesting
+  Longs longs = Longs.instance();
+
+  public AbstractLongAdderAssert(LongAdder longAdder, Class<?> selfType) {
+    super(longAdder, selfType);
+  }
+
+  /**
+   * Verifies that the actual sum has a value in [start, end] range (start included, end included).
+   * <p>
+   * Example:
+   * <pre><code class='java'>
+   * LongAdder actual = new LongAdder();
+   * actual.add(5);
+   *
+   * // assertions succeed
+   * assertThat(actual).hasValueBetween(4, 6)
+   *                   .hasValueBetween(4, 5)
+   *                   .hasValueBetween(5, 6);
+   *
+   * // assertions fail
+   * assertThat(actual).hasValueBetween(6, 8)
+   *                   .hasValueBetween(0, 4);</code></pre>
+   *
+   * @param startInclusive the start value (inclusive).
+   * @param endInclusive   the end value (inclusive).
+   *
+   * @return this assertion object.
+   *
+   * @throws AssertionError if the actual adder is {@code null}.
+   * @throws AssertionError if the actual sum value is not in [start, end] range.
+   * @since 2.16.0 / 3.16.0
+   */
+  public LongAdderAssert hasValueBetween(long startInclusive, long endInclusive) {
+    isNotNull();
+    longs.assertIsBetween(getWritableAssertionInfo(), actual.sum(), startInclusive, endInclusive);
```

Comment:
You can access `info` directly
```suggestion
    longs.assertIsBetween(info, actual.sum(), startInclusive, endInclusive);
```